src/modules/Lan.rb

```diff
@@ -818,154 +818,76 @@ def UseNetworkManager
 
     def IfcfgsToSkipVirtualizedProposal
       skipped = []
-      Builtins.foreach(LanItems.Items) do |current, _config|
-        ifcfg = Ops.get_string(LanItems.Items, [current, "ifcfg"], "")
-        if NetworkInterfaces.GetType(ifcfg) == "br"
+
+      LanItems.Items.each do |_current, config|
+        ifcfg = config["ifcfg"]
+        ifcfg_type = NetworkInterfaces.GetType(ifcfg)
+
+        case ifcfg_type
+        when "br"
           NetworkInterfaces.Edit(ifcfg)
```

Comment:
True. I just adapted the code doing a small cleanup. In general was thinking in use LanItems.IsBridgeable instead of this method as them check almost the same.